main.js

```diff
@@ -0,0 +1,78 @@
+const app = () => {
+    const track = document.querySelector('.track');
+    const play = document.querySelector('.play');
+    const replay = document.querySelector(".replay");
+    const outline = document.querySelector('.moving-time-line circle');
+    const video = document.querySelector('.video-container video');
+    const timeSelect = document.querySelectorAll('.time input');
+
+    const sounds = document.querySelectorAll('.change-sound button');
+    const timeDisplay = document.querySelector('.time-display');
+    const outlineLength = outline.getTotalLength();  
+    
+    let fakeDuration = 600;
```

Comment:
what is initial timeSelect when you open the page? Does it appear only when you click play?